### ISSUE TITLE
[Game Breaking] Mountain Rage Shaker Trinket procs

### ISSUE THREAD

**Issue by SkyboundGamingZerk** [Valhalla-Gaming/Tracker#8495]
**Describe the issue you're having**:
A warrior in a battleground can use this to gain 3100 strength with in a 10 second window and can keep these stacks up for large periods of time. Essentially helping the one shot with shield slam and many other abilities. Please dont make tanks do tons of damage as well as become unkillable :(
**Explain how you expect it work**:
It should have a chance to proc instead of being a 100% chance to proc off the 1 minute cooldown.
**Steps to reproduce the problem**:
----
**Links to Wowhead, YouTube, etc**:
http://www.wowhead.com/item=121806/mountain-rage-shaker
http://www.wowhead.com/npc=99792/elfbane

**Comment by neano300** [comment]
and the trinket have that CD. the time pass really fast in the pvp :D

**Comment by SkyboundGamingZerk** [comment]
Yes it has a cd, but the proc chanced are way higher than its supposed to
be. It's practiacally a 100% chance to proc every minute when attacking. It
shouldnt be like that.

**Comment by GM-Pyromanic** [comment]
The trinket allows 20 stacks of the buff each buff is granted 0.5 seconds for 10 seconds = 20 stacks retail the trinket procs roughly once per 3-4 mob pulls on retail, I will be grabbing the trinket sometime this week to do test on retail with it and report my findings for it.
